packages/application-config/src/validate-config.ts

```diff
@@ -46,3 +48,17 @@ const validateConfig = (
 };
 
 export default validateConfig;
+
+export const validateSubmenuLinks = (
```

Comment:
Nit, if we export multiple things it's better to avoid using `default`. Please adjust the export for `validateConfig` to be `export const validateConfig`.

Also, we can maybe rename the file to `validations.ts` instead of `validate-config.ts`.